whylog/converters/__init__.py

```diff
@@ -32,12 +33,26 @@ def safe_convert(cls, pattern_group):
 
 
 class IntConverter(AbstractConverter):
+    MIN_VALUE = -2000000000
```

Comment:
I am sure you can find a better way to do it utilizing the standard library instead of hardcoding arbitrary integers